src/scene/text-split/SplitBitmapText.ts

```diff
@@ -170,6 +170,9 @@ export class SplitBitmapText extends AbstractSplitText<BitmapText>
             ...config,
         };
 
+        completeOptions.style ??= completeOptions.style || {};
```

Comment:
But this could be simpler:
```suggestion
        completeOptions.style ??= {};
```